File: EventManager/event-management-system/event-management-system.client/src/types/Event.ts
export interface Attendee {
    attendeeId: number;
    name: string;
    email: string;
    eventId: number;
  }
  
  export interface Event {
    eventId: number;
    title: string;
    description: string;
    startDate: string;
    endDate: string;
    location: string;
    attendees: Attendee[];
  }
  
  export interface CreateEventDto {
    title: string;
    description: string;
    startDate: string;
    endDate: string;
    location: string;
    attendees: Attendee[];
  }
  

File: EventManager/event-management-system/event-management-system.Server/Model/Event.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.EntityFrameworkCore;

namespace event_management_system.Server.Model
{
    public class Event
    {


        public int EventId { get; set; }
        public string Title { get; set; }
        public string Description { get; set; }
        public DateTime StartDate { get; set; }
        public DateTime EndDate { get; set; }
        public string Location { get; set; }

        public ICollection<Attendee> Attendee { get; set; }


    }
}